doc/classes/CanvasItem.xml

```diff
@@ -557,6 +557,9 @@
 		<member name="show_behind_parent" type="bool" setter="set_draw_behind_parent" getter="is_draw_behind_parent_enabled" default="false">
 			If [code]true[/code], the object draws behind its parent.
 		</member>
+		<member name="toplevel" type="bool" setter="set_as_toplevel" getter="is_set_as_toplevel">
+			If true, the node will not inherit its transform from parent [CanvasItem]s.
```

Comment:
"true" should be wrapped as `[code]true[/code]` the way it's done for other descriptions for consistency, same for other changes.

Reply:
Fixed